README.md

```diff
@@ -2,8 +2,52 @@
 
 ## Purpose
 
-_... help fill this in!_
+Purpose is *not* to respond to reports of security issues with Node.js, that is
+done by the "security team", see https://github.com/nodejs/node#security
+
+*todo*: should the team be identified in
+https://github.com/nodejs/node#security?
+
+Purpose is to be discussed in working group meetings, but topics raised in the
+WG meeting at NINA16 included:
+
+- Is it worth looking at getting a sec eval of node?
+
+- What is security? Is there definitions that would help us? does CVE/CERT/?
+  have classifications we can use? It would be helpful to have a stance on this
+  so we can more clearly respond to issues that are reported or claimed as
+  "security issues".
+
+Ecosystem:
+
+- sec-wg could take some responsibility for fixing ecosystem issues
+- Best practices on sec problems could use definition and
+  publication/evanglizing, so npmjs.org packages can follow a well-known and
+  robust procedure:
+  - What should an npm module author do if an issue is found in their module?
+  - How should they deal with it?
+  - Should there be a CVE?
+  - How should we (node/the node foundation/sec-wg) communicate with the
+    author?
+  - Its possible there should be a community vulnerability response list, that
+    is available if people want to report a vulnerability in community modules,
+    but don't know how, and do not want it in a public issue/PR on the modules
+    github repo.
```

Comment:
For now I'd suggest something which just captures the high level mandate of the working group (based on an existing example: https://github.com/nodejs/benchmarking)

# Mandate

The Security working group's purpose is to achieve the highest level of security for Node.js and community modules.

Its responsibilities are:

* Work with the node security project to bring community vulnerability data into the foundation as 
  a shared asset
* Set up processes and procedures and follow these to ensure the vulnerability data is updated in 
  an efficient and timely manner.  As a example ensuring there are well documented processes
  for reporting vulnerabilities in community modules.
* Work to set a high standard for the Node.js project.  Possibly efforts could include penetration 
  testing, security reviews etc, review guidelines, coding standards etc.
* Review and recommend processes for handling of security reports (but not the implementation of those processes)_